### ISSUE TITLE
Now Migration: Size inline images

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#6019]
Inline images are being brought over properly, but the sizing isn't matching the source (instead, they are all defaulting to medium).

There should be some logic to handle this, but need to check how it was sized in the source and map that to the appropriate size in the destination and/or fix the logic that is supposed to handle this.

Additionally, reconsider what the default image size should be set to in cases where the above doesn't work. Small may be more appropriate than medium).

While working on this, take out the extra `<br>` that's getting added to captions, also noted in this comment.
https://github.com/uiowa/uiowa/pull/5941#issuecomment-1363269916

It looks like both the `<br>` and the missed sizing may be due to the media existing inside a wrapper div which contains sizing/alignment classes. The classes are missed because they're not directly part of the media, and the extra div might be causing the added spacing after the media has been converted.

Continue work in https://github.com/uiowa/uiowa/pull/5941

**Comment by cory-skeers** [comment]
I worked some on this alongside #6060. 

In the source, this is handled by the wrapper class like:
```
<div class="video-right-320">[[{"fid":"13636","view_mode":"iowanow_medium","fields":{"format":"iowanow_medium"},"type":"media","attributes":{"alt":"Iowa Re-Designated As NCI Cancer Center","class":"media-element file-iowanow-medium"}}]]<br />
	George Weiner, director of the Holden Comprehensive Cancer Center, discusses the center&#39;s continued designation from the National Cancer Institute.</div>
```
As part of the migration, we were already grabbing images in patterns like this in order to fix captions, and I've since expanded that to a) hit both videos and images and b) update the resultant `drupal-media` element with corresponding alignments and sizes/view-modes.

There are defaults in place as part of this replacement, as well as the original defaults should the replacement process not work, but in looking through values on the source, it seems pretty consistent across the board.

In the same section, I also added some spacing cleanup (replacing line breaks with spaces, and then replacing multiple spaces with single spaces and trimming). This could potentially affect content, but in all the examples I've looked at, line breaks were added for visual spacing only, and could safely be removed. In the case they were used in a caption to separate content (like in an address, or list of names), the updated spacing might not be an exact match to the source, but should still be a good-faith representation (eg a full address listed on one line, or a list of names that are all on one line but do not run together).
